apps/support/Makefile.inc

```diff
@@ -103,6 +106,8 @@ CXX-host-metal ?= $(CXX)
 CXX-host-hvx_128 ?= $(CXX)
 CXX-host-hvx_64 ?= $(CXX)
 CXX-host-hvx ?= $(CXX)
+CXX-hexagon-32-qurt-hvx ?= hexagon-clang++
+CXX-hexagon-32-qurt-hvx_128 ?= $(CXX-hexagon-32-qurt-hvx)
```

Comment:
I think we should not add this `hvx_128` version of these flags either? I think the only non-deprecated thing is `hvx` right?